test/SlowTests/Server/Documents/ETL/Elasticsearch/ElasticsearchEtlTests.cs

```diff
@@ -0,0 +1,871 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+using Nest;
+using Newtonsoft.Json.Linq;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Operations.ConnectionStrings;
+using Raven.Client.Documents.Operations.ETL;
+using Raven.Client.Documents.Operations.ETL.Elasticsearch;
+using Raven.Client.ServerWide.Operations.Certificates;
+using Raven.Server.Documents.ETL.Providers.Elasticsearch;
+using Raven.Server.Documents.ETL.Providers.Elasticsearch.Test;
+using Raven.Server.NotificationCenter.Notifications;
+using Raven.Server.ServerWide.Context;
+using Raven.Tests.Core.Utils.Entities;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Server.Documents.ETL.Elasticsearch
+{
+    public class ElasticsearchEtlTests : EtlTestBase
+    {
+        public ElasticsearchEtlTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        protected const string defaultScript = @"
+var orderData = {
+    Id: id(this),
+    OrderLinesCount: this.OrderLines.length,
+    TotalCost: 0
+};
+
+for (var i = 0; i < this.OrderLines.length; i++) {
+    var line = this.OrderLines[i];
+    var cost = (line.Quantity * line.PricePerUnit) *  ( 1 - line.Discount);
+    orderData.TotalCost += line.Cost * line.Quantity;
+    loadToOrderLines({
+        OrderId: id(this),
+        Qty: line.Quantity,
+        Product: line.Product,
+        Cost: line.Cost
+    });
+}
+
+loadToOrders(orderData);
+";
+
+        [Fact]
+        public void Simple_script()
+        {
+            using (var store = GetDocumentStore())
+            {
+                SetupElasticEtl(store, defaultScript, new List<string>() {"Orders", "OrderLines"});
+                var etlDone = WaitForEtl(store, (n, statistics) => statistics.LoadSuccesses != 0);
+
+                using (var session = store.OpenSession())
+                {
+                    session.Store(new Order
+                    {
+                        OrderLines = new List<OrderLine>
+                        {
+                            new OrderLine {Cost = 3, Product = "Cheese", Quantity = 3}, new OrderLine {Cost = 4, Product = "Bear", Quantity = 2},
+                        }
+                    });
+                    session.SaveChanges();
+                }
+
+                var node = new Uri("http://localhost:9200");
```

Comment:
Let's move the connection to a constant or some known location, rather than in here